### ISSUE TITLE
Access counter variable

### ISSUE THREAD

**Issue by rayfranco** [kazupon/vue-i18n#67]
When we use pluralization, we use the counter itself in the sentence.
To avoid repeating, we could inject a predefined variable for the counter in `$tc` method.

For example with this translation:

``` json
"messages": {
  "counter": "You clicked { count } time | You clicked { count } times"
}
```

Instead of doing this:

``` javascript
<p>$tc('messages.counter', userClicks, { count: userClicks })</p>
```

You could do this (by assuming `count` is used as the predefined variable)

``` javascript
<p>$tc('messages.counter', userClicks)</p>
```

**Comment by popod** [comment]
Will this be implemented ? This should be very useful !

**Comment by exoego** [comment]
@popod @rayfranco 
This was finally implemented in v8.2.0.
https://kazupon.github.io/vue-i18n/guide/pluralization.html#accessing-the-number-via-the-pre-defined-argument
